static/index.html

```diff
@@ -105,7 +105,7 @@ <h3>Drupal in Cebu</h3>
                 <h3>Drupal on Twitter</h3>
                 <p> <i class="fab fa-twitter"></i><a href="https://twitter.com/DrupalCampCebu" target="_blank">@DrupalCampCebu.</a> Who said conferences are not funny? More photos from our last Drupalcamp Cebu: <a href="https://www.facebook.com/media/set/?set=oa.723538837835150&type=3" target="_blank">https://www.facebook.com/media/set/?set=oa.723538837835150&type=3</a><a href="https://twitter.com/hashtag/drupalcampcebu?src=hash" target="_blank"> #drupalcampcebu</a></p>
                 <p> <i class="fab fa-twitter"></i><a href="https://twitter.com/DrupalCampCebu" target="_blank">@DrupalCampCebu.</a> Drupalcamp Cebu 2017 group photo. This was our largest Drupalcamp :) <a href="https://twitter.com/hashtag/drupalcampcebu?src=hash" target="_blank">#drupalcampcebu</a> <a href="https://twitter.com/hashtag/drupalcamp?src=hash" target="_blank">#drupalcamp</a></p>
-                <p> <i class="fab fa-twitter"></i><a href="https://twitter.com/DrupalCampCebu" target="_blank">@DrupalCampCebu.</a> Drupalcamp trainer @CasisMary published her full report from the event. Very interesting read with testimonials from attendees! Thanks Mary! <a href="https://www.prometsource.com/blog/drupalcamp-cebu-2017-review" target="_blank">prometsource.com/blog/drupalcamp-cebu-2017-review</a> #drupalcamp #drupalcampcebu</p>
+                <p> <i class="fab fa-twitter"></i><a href="https://twitter.com/DrupalCampCebu" target="_blank">@DrupalCampCebu.</a> Drupalcamp trainer @CasisMary published her full report from the event. Very interesting read with testimonials from attendees! Thanks Mary! <a href="https://www.prometsource.com/blog/drupalcamp-cebu-2017-review" target="_blank">prometsource.com/blog/drupalcamp-cebu-2017-review</a> <a href="https://twitter.com/hashtag/drupalcamp?src=hash" target="_blank">#drupalcamp</a> <a href="https://twitter.com/hashtag/drupalcampcebu?src=hash" target="_blank">#drupalcampcebu</a></p>
               </div>
             </div>
             <div class="col-sm-12 col-md-12 col-lg-6 px-0">
```

Comment:
<div class="col-sm-12 col-md-12 col-lg-6 px-0">

I think you don't have to declare col-md-12 here :)
col-sm-12 will be carried over to bigger breakpoint UNLESS you set it again, in this case 'col-lg-6'. 
col-md is by default 12 columns as well since your col-sm is 12.